<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrgComboBoxPopup</class>
 <widget class="QWidget" name="OrgComboBoxPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabOrg">
      <attribute name="title">
       <string>&amp;Организации</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="QListView" name="tblOrg">
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSearch">
      <attribute name="title">
       <string>&amp;Поиск</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="6" column="2">
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
         </property>
         <property name="centerButtons">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>&amp;Название содержит</string>
         </property>
         <property name="buddy">
          <cstring>edtName</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblOGRN">
         <property name="text">
          <string>&amp;ОГРН</string>
         </property>
         <property name="buddy">
          <cstring>edtOGRN</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="edtINN"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblINN">
         <property name="text">
          <string>&amp;ИНН</string>
         </property>
         <property name="buddy">
          <cstring>edtINN</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="edtOKATO"/>
       </item>
       <item row="6" column="0" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>172</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblOKATO">
         <property name="text">
          <string>ОКАТО</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="edtOGRN"/>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblIsHospital">
         <property name="text">
          <string>ЛПУ</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="cmbIsMedical">
         <item>
          <property name="text">
           <string>не определено</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>поликлиника</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>стационар</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>прочая мед.организация</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>СМП</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tblOrg</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>cmbIsMedical</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtINN</tabstop>
  <tabstop>edtOGRN</tabstop>
  <tabstop>edtOKATO</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
